modules/base/mysql.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker-compose exec -e COLUMNS="`tput cols`" -e LINES="`tput lines`" -e SHELL=bash -u 0 cli mysql -uroot -p${MYSQL_ROOT_PASSWORD} -hmysql "${@:2}"
```

Comment:
You can use here directly the mysql container.

Reply:
I could yes. Are there volumes bound to the mysql container? So I can import files via `source foobar.sql`. This is possible with the cli container.

Reply:
Ahh okay i understand